subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -9042,6 +9050,15 @@ void radio_rx_dequeue(void)
 		LL_ASSERT(0);
 		break;
 	}
+
+	if (radio_pdu_node_rx->hdr.type == NODE_RX_TYPE_CONNECTION) {
+		u8_t bm = ((u8_t)_radio.scanner.is_enabled << 1) |
+			  _radio.advertiser.is_enabled;
+
+		if (!bm) {
+			ll_adv_scan_state_cb(0);
```

Comment:
No, shall not, its a consequence of connection complete, and it shall only be generated after advertiser and/or scanner have been stopped in ISR.